cafebabel/tests/test_base.py

```diff
@@ -1,8 +1,16 @@
+from http import HTTPStatus
 
-def test_homepage_is_displaying(client):
+
+def test_homepage_is_redirecting_to_default_language(client):
```

Comment:
Isn't it quite the same as `test_homepage_is_redirecting_unknown_language_to_default`?

Reply:
`test_homepage_is_redirecting_to_default_language` is testing how the homepage will display by default.
`test_homepage_is_redirecting_unknown_language_to_default` will send an unknown _Accepted language_ by the browser to the server and check how the server is handling.

But if it's confusing to you it may also mean that the names are not consistent.
Any suggestion?

Reply:
It's clearer now, let's go for it 👍